tools/autograd/templates/VariableType.h

```diff
@@ -4,6 +4,8 @@
 
 #include <ATen/ATen.h>
 
+#include "torch/csrc/WindowsTorchApiMacro.h"
```

Comment:
nit: prefer <> in the future

Reply:
ok, i'll switch it. what's the guideline to pick one or the other? It seems that quotes are more common in the codebase

```
(venv)[andersj@devgpu201.prn2 ~/local/pytorch] git grep 'include "torch'|wc
    911    1822   69467
(venv)[andersj@devgpu201.prn2 ~/local/pytorch] git grep 'include <torch'|wc
    238     479   16250
```

Reply:
I will make a codemod soon! We just had to change some includes that were not absolute paths (in which case <> is incorrect), but now I'll make that codemod